package/src/skia/types/Skia.ts

```diff
@@ -39,7 +39,6 @@ export interface SkiaApi {
   RRectXY: (rect: SkRect, rx: number, ry: number) => SkRRect;
   RSXform: (scos: number, ssin: number, tx: number, ty: number) => SkRSXform;
   Color: (color: Color) => SkColor;
-  parseColorString: (color: string) => SkColor | undefined;
```

Comment:
this was not exported on the C++ side but was not caught because Skia didn't implement this type.